src/main/cpp/subsystems/SwerveModuleSubsystem.cpp

```diff
@@ -58,6 +58,8 @@ SwerveModuleSubsystem::SwerveModuleSubsystem(
 
   frc::SmartDashboard::PutNumber("KFF Input " + std::to_string(m_driveMotor->GetDeviceId()), ModuleConstants::kFFModuleDriveController);
   frc::SmartDashboard::PutNumber("Wheel Offset " + std::to_string(m_turningMotor->GetDeviceId()), ModuleConstants::wheelOffset);
+
```

Comment:
Probably a good reason for it but the tenth second sleep functions sprinkled in?

Reply:
To delay some can usage between setting up motors, hopefully for getting rid of the wheel randomly flipping issue.